client/src/connection/itc/const.ts

```diff
@@ -0,0 +1,30 @@
+// Copyright © 2024, SAS Institute Inc., Cary, NC, USA.  All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import { l10n } from "vscode";
+
+export const humanReadableMessages = [
```

Comment:
Hey @clangsmith . Do we think it's worth having a resource to expand on what these error messages mean? Or perhaps we can tweak these messages to elaborate? 

Really, the only messages that look like they _could_ require more explanation are the errors on line 18 and 28 (imo).